frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx

```diff
@@ -8,7 +8,7 @@ import { AppearanceSettings } from "./AppearanceSettings";
 import type { OverviewSettingsProps } from "./OverviewSettings";
 import { OverviewSettings } from "./OverviewSettings";
 
-const TABS = {
+export const EMBED_MODAL_TABS = {
```

Comment:
Changed the name to something more descriptive since I need to use them in other files